.github/workflows/appengine-typescript.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       pull-requests: 'write'
       id-token: 'write'
     steps:
-    - uses: 'google-github-actions/auth@v0.3.1'
+    - uses: 'google-github-actions/auth@v0.6.0'
```

Comment:
I think the recent version of `google-github-actions/auth` asks you to first checkout the source code then use the `auth`.